src/components/LogRow/RowRenderer/index.tsx

```diff
@@ -17,14 +17,14 @@ const ParsleyRow: RowRendererFunction = ({ logType, processedLogLines }) => {
     expandLines,
     getLine,
     getResmokeLineColor,
-    scrollToLine,
     preferences,
     range,
+    scrollToLine,
     searchLine,
     searchState,
   } = useLogContext();
   const { searchTerm } = searchState;
-  const { wrap, prettyPrint } = preferences;
+  const { prettyPrint, wrap } = preferences;
```

Comment:
I was able to change the order of a destructured variable and commit without any warning/error appearing. I think we should be surfacing this rule at least during the lint-staged process. Are you able to reproduce?

Reply:
I don't think this rule actually affects destructured objects. I'll re-add the other rule to make this work.